lib/styles/colors.dart

```diff
@@ -0,0 +1,5 @@
+import 'package:flutter/material.dart';
+
+const primary = Colors.green;
```

Comment:
We already have constants/custom_colors.dart
Can we combine these two files?

Reply:
I didn't notice those, and yes we should put the theme info in a single file.

Reply:
I would set the primary color to 
`  static const Green = Color.fromRGBO(46, 181, 146, 1);`

Reply:
resolved in https://github.com/7flash/flutter_seeds_wallet/pull/46/commits/b2c035db11f41859c570e58ec2272f5bc577bd33